src/openrct2/management/Award.cpp

```diff
@@ -305,9 +305,9 @@ static bool award_is_deserved_best_food(int32_t activeAwardTypes)
         auto rideEntry = get_ride_entry(ride.subtype);
         if (rideEntry != nullptr)
         {
-            if (!(shopTypes & (1ULL << rideEntry->shop_item[0])))
+            if (!(shopTypes & EnumToFlag(rideEntry->shop_item[0])))
```

Comment:
I'm surprised that ShopTypes is `uint32_t` it should be `uint64_t`. Investigate if this should be changed or leave a `TODO`